### PR TITLE
Add possibility to have just relative urls in image names

### PR DESCRIPTION
**Pull request by danielhusar**
Hi I think its a good idea, as this allow you to compare some local url like `http://localhost:3000` with some live urls :)

@stefanjudis what you think about this?

### CODE REVIEW

File: tasks/lib/photobox.js [hunk 1]
```diff
@@ -505,10 +505,15 @@ PhotoBox.prototype.startPhotoSession = function() {
   this.pictures.forEach( function( picture ) {
     this.grunt.log.writeln( 'started photo session for ' + picture );
 
+    var parsedImage = require( 'url' ).parse(picture);
```
Comment (danielhusar) [inline tasks/lib/photobox.js]: I cant use `require( 'url' )` inside of child process, so adding it here to pass image name together with url :(

**Comment by stefanjudis** [comment]
@danielhusar Oh man, something like this is on my todo list for a really long time already.

I'll try to check it on the weekend. :)
If not feel free to ping me again.

Thanks a lot for the effort. :bowtie:

**Comment by danielhusar** [comment]
Cool, thanks :)

**Comment by danielhusar** [comment]
Hey, any chance to look on it yet ? :)

**Comment by stefanjudis** [comment]
Aaaah great reminder. ;)

Not really, sorry.
But I don't mind merging it. :)

You agreed to become a maintainer, right?

No need also for working in your fork - you can directly branch in this repo.

File: README.md [hunk 2]
```diff
@@ -82,6 +82,15 @@ Default value: `[ 'http://google.com' ]`
...
+#### options.relativePaths
+
+Type: `Boolean`
+
+Default value: `false`
+
+If final images should contain only realative urls.
+
+
 #### options.userName
```
Comment (stefanjudis) [inline README.md]: Just had a quick look - but I'm not sure I get it completely how this works. 

Maybe we can add another example at the end of the readme explaining a bit more? :bowtie:

File: tasks/lib/photobox.js (hunk 1 above)
Comment (stefanjudis) [inline tasks/lib/photobox.js]: i'm really into spaces - so do you mind to change to `( picture )`? :blush:

**Comment by stefanjudis** [comment]
Just had a super quick look. :)

Can you extend `readme` a bit more and stick to coding style ( spaces ) on several places.
I'll try to make functional check asap.

But as said - I've got no problem to merge and publish. If you say 'GO'. :)

Thanks a lot for taking time and your contribution. :)

**Comment by danielhusar** [comment]
Hey yeah cool, will add some more examples tomorrow, thanks for quick look ! :)

**Comment by danielhusar** [comment]
@stefanjudis I have polished spacings a little, hopefully its better now :) and added some sample config to readme. (I know the described usage scenario there is not perfect, but this way you can very easily compare two version of same page, but maybe we should introduce something like base urls and generate screens for all urls and then generate index.html ? )

**Comment by danielhusar** [comment]
Ah this getting more and more complicated because I forgot about image magic.. I will fix test later today :)

**Comment by danielhusar** [comment]
Hey :) So I have tested it yesterday for quite long and it looks like all is working fine, so I can merge it now, and you can publish new version when you have some time :)

**Comment by danielhusar** [comment]
Also, what you think about moving to something more robust to generate images like [pageres](https://github.com/sindresorhus/pageres) ?

**Comment by stefanjudis** [comment]
@danielhusar nothing against pageres! If you want to put it in, go ahead. :)

**Pull request merged**